### PR TITLE
WIP, one hard error left

### PR DESCRIPTION
**Pull request by marcoabbrancati**
WIP

### CODE REVIEW

**Comment by marcoabbrancati** [comment]
tl;dr
tried out to import the missing classes and analysed the testWrite-method, but couldnt determine, why the tests fail.

conclusion:
using /srv/git/SessionHandler/vendor/bin/phpunit --bootstrap bootstrap.php .
two errors are triggered
     1) Horde\SessionHandler\Storage\BuiltinTest::testGc 
         Caused by
         ErrorException: include(PHPUnit/Framework/TestFailure.php): failed to open stream: No such file or directory in /srv/git       /SessionHandler/vendor/horde/test/lib/Horde/Test/Autoload.php:59

     2) Horde\SessionHandler\Storage\Sql\Pdo\SqliteTest::testWrite
        Horde_Test_Exception: The "Horde_Db_Adapter_Pdo_Sqlite" class is unavailable!

if I use 'horde-components qc unit' instead, there are six errors triggered, all of them because **BaseTestCase wasnt found in ../Storage/BuiltinTest.php:18**

**Comment by ralflang** [comment]
Have you installed horde/db in your uut dir before running the test?

**Comment by marcoabbrancati** [comment]
adapted ci.yml to install all necessary packages, but the error explained above remains

**Comment by marcoabbrancati** [comment]
at least one error remained:
1) Horde\SessionHandler\Storage\BuiltinTest::testGc
PHPUnit\Framework\Exception: [...]
Caused by
ErrorException: include(PHPUnit/Framework/TestFailure.php): failed to open stream: No such file or directory in /srv/git/SessionHandler/vendor/horde/test/lib/Horde/Test/Autoload.php:59
Stack trace:
#0 /srv/git/SessionHandler/vendor/horde/test/lib/Horde/Test/Autoload.php(59): PHPUnit\Util\PHP\AbstractPhpProcess::PHPUnit\Util\PHP\{closure}()

commenting out the function didnt bring any effort.
